ev3dev2/motor.py

```diff
@@ -2123,19 +2108,6 @@ def follow_line(self,
 
         self.stop()
 
-    def calibrate_gyro(self):
-        """
-        Calibrates the gyro sensor.
-
-        NOTE: This takes 1sec to run
-        """
-        assert self._gyro, "GyroSensor must be defined"
-
-        for x in range(2):
-            self._gyro.mode = 'GYRO-RATE'
-            self._gyro.mode = 'GYRO-ANG'
-            time.sleep(0.5)
-
```

Comment:
This method was not present in 2.0.0 so we are ok here
https://github.com/ev3dev/ev3dev-lang-python/blob/ev3dev-stretch/2.0.0/ev3dev2/motor.py